docs/tutorials/containermonitoringaddon.md

```diff
@@ -113,6 +113,8 @@ For upgrading the container monitoring addon, you can disable the monitoring add
 
 Starting with agent version ciprod05262020, chart will support Container Runtime Interface(CRI) compatiable runtimes such as CRI-O and ContainerD etc. in addition to Docker/Moby.
```

Comment:
please change to "container monitoring add-on" instead of chart here.

Reply:
updated.